foods.html

```diff
@@ -5,12 +5,13 @@
   <title>Quantified Self - Manage Foods</title>
 </head>
 <body>
-
+  <h3>Manage Foods</h3>
   <div>
     <form class="new-food-form">
       <span class="food-labels">Name</span><br>
       <input type="text" name="food-name" size="40" class="food-form"><br>
       <span class="food-placeholders">Please enter a food name</span><br>
+      <span class="error" id="msg1">Please add a name</span><br>
```

Comment:
This error alert is redundant, but I like it better than the current solution.  I'm going to make a waffle card to remove the old functionality and style the new alerts.